Makefile

```diff
@@ -7,22 +7,22 @@ check-default:
 
 # Checks the source code with all features enabled.
 check-mixed:
-	cargo check --all-features
-	cargo check --all-features --tests
+	cargo clippy --all-features
+	cargo clippy --all-features --tests
```

Comment:
There's one more `cargo check` that needs to be updated, L6 under the `check-default` recipe

Reply:
Done in 70dc957